app/src/main/java/com/xa/xpensauditor/AddTransactionActivity.java

```diff
@@ -1120,7 +1120,7 @@ protected void onCreate(Bundle savedInstanceState) {
         activity = this;
 
         mRootRef = new Firebase("https://xpensauditor-g11-default-rtdb.firebaseio.com/");
-
+        String test_string;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.